pov_tools/create_POV_data_files

```diff
@@ -0,0 +1,218 @@
+#!/bin/bash
+
+# create_POV_data_files
+
+cd "$(dirname "$0")"
+clear
+echo ""; echo "Welcome! Let's make some POV images!"; echo ""
+# replace whitespace in source filename with underscores
+find . -type f -name "* *" | while read file; do mv "$file" ${file// /_}; done
+have_logo=0
+height_only=0
+preset=0
+# get quantity of .png files and build array to print if multiples
+files=$(ls 2>/dev/null -Ubad1 -- *.png | wc -l)
```

Comment:
-U (sorting) is not needed, it's also not POSIX, so it might not work with all "ls" programs.
-b (quoting) is also not needed, nor POSIX
-a is for listing .-files, which is not needed

2>/dev/null is also not needed afaik.

Reply:
gone.